src/client/src/rt-types/trade.ts

```diff
@@ -11,6 +11,7 @@ export interface Trade {
   valueDate: Date
   status: any
   highlight?: boolean
+  latest?: boolean
```

Comment:
We are extending the data model with properties only required for testing, I am not sure this is ideal?

Ideally if we can restrict testing annotations to just the data-qa attributes that would be preferable.